addons/account/views/report_invoice.xml

```diff
@@ -142,7 +143,7 @@
                                                 <span class="text-nowrap" t-field="line.discount">0</span>
                                             </td>
                                             <t t-set="taxes" t-value="', '.join([(tax.invoice_label or tax.name) for tax in line.tax_ids])"/>
-                                            <td name="td_taxes" t-attf-class="text-start {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }} {{ 'text-nowrap' if len(taxes) &lt; 10 else '' }}">
+                                            <td name="td_taxes" t-attf-class="text-start {{ 'd-none' if not show_taxes else '' }} {{ 'd-none d-md-table-cell' if report_type == 'html' and show_taxes else '' }} {{ 'text-nowrap' if len(taxes) &lt; 10 else '' }}">
```

Comment:
It would be nice to not compute `taxes` if you don't display it, so put a div or something that includes this line and the one above

Reply:
What do you think if I did that on them all?
I like wrapping the tax field inside div with condition. I think it will be less complex comparing to large condition like that.
`{{ 'd-none' if not show_taxes else '' }} {{ 'd-none d-md-table-cell' if report_type == 'html' and show_taxes else '' }}`

Or what do you think?